src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -27,39 +27,40 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
 }) => {
   const dispatch = useDispatch();
   return (
-    <div className="schedule-item-container">
-      <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
-        <div>
-          <b>{formatUtcSeconds(event.start_utc_seconds)}</b>
-        </div>
-        <div>
+    <div className="ScheduleItem">
+      <div
+        className={`ScheduleItem__time-section ${
+          isCurrentEvent ? "ScheduleItem__primary" : ""
+        }`}
+      >
+        <span className="ScheduleItem__event-date">
+          {labelDayRoomSchedule(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
+          {formatUtcSeconds(event.start_utc_seconds)}
+        </span>
+        <span className="ScheduleItem__event-time">
           {formatUtcSeconds(
             event.start_utc_seconds + event.duration_minutes * 60
           )}
-        </div>
+        </span>
       </div>
-      <div className="event-section">
-        <div>
-          <div className={`${isCurrentEvent ? "primary" : ""}`}>
-            <div>
-              <b>{event.name}</b>
-            </div>
-            <div>
-              by <b>{event.host}</b>
-            </div>
-          </div>
-          <div className="event-description">
+      <div className="ScheduleItem__event-section">
+        <div className={`${isCurrentEvent ? "ScheduleItem__primary" : ""}`}>
```

Comment:
classnames

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/05e69b6ea9cffce574fb944d32462f8f82570d45